packages/web-components/src/components/video-player/video-player-composite.ts

```diff
@@ -194,7 +194,7 @@ class DDSVideoPlayerComposite extends HybridRenderMixin(
   /**
    * `true` to autoplay, mute, and hide player UI.
    */
-  @property({ type: Boolean, attribute: 'background-mode' })
+  @property({ type: Boolean, attribute: 'background-mode', reflect: true })
```

Comment:
I think we should also reflect this as a Boolean attribute in the stories. Many instances currently read as `background-mode="true"`, eg. https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/be33a5b6c220e776dc9571dab345bd36043190b7/packages/web-components/src/components/leadspace/__stories__/leadspace.stories.ts#L173

The reflection seems to force the changes from `background-mode="true"` to just `background-mode`, so it's not a huge deal, just code style at this point.

Reply:
Addressed in recent commit https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/12081/commits/5fc572f5b8c2781e6ff8931ebdd60a8f433a2f44